<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDos</class>
 <widget class="QWidget" name="FormDos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP4</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Downloads/icons8-carpeta-de-películas-96.png</normaloff>../Downloads/icons8-carpeta-de-películas-96.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject{
	background-color: #555;
}</string>
  </property>
  <widget class="QLabel" name="label_titulo_mp4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: white;
background-color: #ccc;
border: 2px solid #555;</string>
   </property>
   <property name="text">
    <string>Descarga MP4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_link_mp4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Insertar el Link de click y espere el mensaje:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_link_mp4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_descargar_mp4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Descargar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: white;
border: 2px solid #555;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
